pubsub/lib/pubsub_samples.ex

```diff
@@ -51,4 +77,106 @@ defmodule GoogleApi.PubSub.Samples do
 
     "deleted #{topic_name}"
   end
+
+  @doc """
+  Get a PubSub subscription.
+
+  ## Examples
+
+      iex> GoogleApi.PubSub.Samples.get_subscription("YOUR_PROJECT_ID", "test-subscription")
+
+  """
+  def get_subscription(project_id, subscription_name) do
+    # Authenticate
+    {:ok, token} = Goth.Token.for_scope("https://www.googleapis.com/auth/cloud-platform")
+    conn = GoogleApi.PubSub.V1.Connection.new(token.token)
+
+    # Make the API request.
+    response = GoogleApi.PubSub.V1.Api.Projects.pubsub_projects_subscriptions_get(
+      conn,
+      project_id,
+      subscription_name
+    )
+
+    case response do
+      {:ok, subscription} -> subscription
+      {:error, _} -> nil
+    end
+  end
+
+  @doc """
+  Subscribe to a PubSub topic, print the message on push.
+
+  ## Examples
+
+      iex> GoogleApi.PubSub.Samples.create_subscription("YOUR_PROJECT_ID", "test-topic", "test-subscription")
+      "created subscription projects/YOUR_PROJECT_ID/subscriptions/test-subscription on topic projects/YOUR_PROJECT_ID/subscriptions/test-topic"
+
+  """
+  def create_subscription(project_id, topic_name, subscription_name) do
+    # Authenticate
+    {:ok, token} = Goth.Token.for_scope("https://www.googleapis.com/auth/cloud-platform")
+    conn = GoogleApi.PubSub.V1.Connection.new(token.token)
+
+    # Make the API request.
+    {:ok, response} = GoogleApi.PubSub.V1.Api.Projects.pubsub_projects_subscriptions_create(
+      conn,
+      project_id,
+      subscription_name,
+      [body: %{
+        :topic => "projects/#{project_id}/topics/#{topic_name}"
+      }]
+    )
+
+    "created subscription #{response.name} on topic #{response.name}"
```

Comment:
It's a bit bizarre that we're returning this string rather than displaying it. And that it's ultimately being used (pubsub.ex line 23) as a boolean. Can we clean this up a bit? Display the line here (and elsewhere, in create_topic and delete_topic above), and just return `true` or `false` if that's what we want.

Reply:
Same for other functions that participate in this logic, like get_subscription.

Reply:
fixed